tests/queries/1_stateful/00178_random_sample_from_array.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# shellcheck disable=SC2154
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# Initialize variables
+total_tests=0
+passed_tests=0
+
+# Test randomSampleFromArray with integer array
+query_result=$(clickhouse-client -q "SELECT randomSampleFromArray([1,2,3], 2)")
+if [[ "$query_result" =~ "1" ]] || [[ "$query_result" =~ "2" ]] || [[ "$query_result" =~ "3" ]]; then
```

Comment:
So, if the function happens to return `[1]`, this will count as passed, right? In this case, better test against all three possible result arrays `[1,2]`, `[1,3]` and `[2,3]`.

Reply:
The tests had a bug; the checks were incorrect, so I fixed it. You now should see different checks.